core/src/scene/sceneLoader.cpp

```diff
@@ -1066,6 +1066,11 @@ void SceneLoader::loadSource(const std::shared_ptr<Platform>& platform, const st
         isMBTilesFile = urlLength > extLength && (url.compare(urlLength - extLength, extLength, extStr) == 0);
     }
 
+    bool isTms = false;
+    if (auto tmsNode = source["tms"]) {
```

Comment:
Mmmm we've been back and forth on this with YAML Nodes; I think the conclusion we reached is that Nodes should be passed by value, since they themselves are just a reference to underlying data.